src/util/ancestors.ts

```diff
@@ -0,0 +1,6 @@
+import { equalThoughtRanked } from './equalThoughtRanked'
+import { Path,Child } from '../types'
+
+
```

Comment:
Extra blank line. Can you make sure the typescript linter is picking this up?

Reply:
We can add a config for picking up such issues. If it's a long process, we can tackle it in a different issue.